pod/perlrecharclass.pod

```diff
@@ -795,7 +795,7 @@ Perl recognizes the following POSIX character classes:
  alpha  Any alphabetical character (e.g., [A-Za-z]).
  alnum  Any alphanumeric character (e.g., [A-Za-z0-9]).
  ascii  Any character in the ASCII character set.
- blank  A GNU extension, equal to a space or a horizontal tab ("\t").
+ blank  A space or a horizontal tab ("\t").
```

Comment:
AIUI locales can define additional character, but space and tab are just the ones that are guaranteed to be there, so using the formulation like the others would be more correct.

```suggestion
 blank  Any blank character (e.g., space or horizontal tab ("\t")).
```

Reply:
I've revised the patch.